nimble/host/syscfg.yml

```diff
@@ -440,6 +440,16 @@ syscfg.defs:
         description: >
             Sysinit stage for the NimBLE host.
         value: 200
+    BLE_HS_OWN_MBUF_FOR_HEADERS:
+        description: >
+             In order to not waste big msys blocks for the ATT + L2CAP and HCI headers
+             with this option on there will be dedicated mempool for this.
+        value : 0
+    BLE_HS_OWN_MBUF_FOR_HEADERS_CNT:
+        description: >
+            Number of blocks for the headers. If this will be used up, MSYS block
+            will be used.
+        value: 'MYNEWT_VAL_MSYS_1_BLOCK_COUNT'
```

Comment:
you can combine both into single option (>0 means use separate pool)
I'd call it e.g. `BLE_HS_L2CAP_HDR_POOL_COUNT

Reply:
agree